### ISSUE TITLE
Localization into Russian.

### ISSUE THREAD

**Issue by TheFuzzStone** [keybase/client#23969]
Hello community!

I've been looking for some community manager in Keybase chat (keybasefriends) for a while, but never found one.

Not so long ago the [Keybase Book](https://book.keybase.io/) was presented to Keybase community.

I have a question, is the community interested in localization of information on this site into Russian? If the answer is yes, where can I apply for funding for a translation team?

I know that Keybase stopped distributing XLM tokens because of cheaters (fake accounts) and I think it was the right decision. Our team discussed and decided to offer the Keybase project help in localization into Russian not only of the Keybase Book, but also of other parts of the Keybase ecosystem (community decides what should be translated), and thus introduce Keybase to Russian-speaking users.

---

### About our team

We are Monero enthusiasts from Ukraine and Russia.
On [our forum](https://xmr.ru/) you can find a lot of stuff that has been translated from English into Russian, including announcements, news, wallets manuals and other technical documentation related to the Monero project, like: 

* [Monero Research Lab papers](https://web.getmonero.org/resources/research-lab/) ---> ([Russian version](https://docs.xmr.ru/research/))
* [Breakin Monero](https://www.monerooutreach.org/breaking-monero/) ---> ([Russian version](https://docs.xmr.ru/breaking/))
* [Monero Konferenco](https://www.monerooutreach.org/monero-konferenco/) ---> ([Russian version](https://docs.xmr.ru/konferenco/))
* [Critical Decentralisation Cluster (36c3)](https://www.youtube.com/watch?v=Xu_QH6oi7oA&list=PLsSYUeVwrHBn07zTBg7fGHRW5Kn_Z3FJL) ---> ([Russian version](https://docs.xmr.ru/cdc/))
* Monero books:
        * [Mastering Monero](https://masteringmonero.com/) ---> ([Russian version](https://docs.xmr.ru/books/mastering-monero/)) ([Some changes are currently under way due to the release of an updated version of the book](https://github.com/monerobook/monerobook/pull/81))
        * [Zero to Monero](https://www.getmonero.org/library/Zero-to-Monero-1-0-0.pdf) ---> ([Russian version](https://docs.xmr.ru/books/zero-to-monero/))

Also our team translated the official [Monero GUI wallet](https://github.com/monero-project/monero-gui) into Russian and Ukrainian, as well as mobile [Monerujo](https://github.com/m2049r/xmrwallet) (Android) and [Cake Wallet](https://github.com/fotolockr/CakeWallet) (Android/iOS). 

Every document is thoroughly checked within our team. 
If I do a translation, then another member (an editor) of the team checks that the translation is correct and then another member (a corrector) checks for punctuation and stylistic errors.

Our team members: @v1docq47, @LvMsterfild, @TheFuzzStone, our corrector doesn't have a Github.

---

### About me:

I'm privacy and crypto enthusiast. Co-founder of the biggest local Monero community: [XMR.RU](https://xmr.ru/) (this is a non-profit project). 

My PGP-key, Keybase, Bitcointalk, Twitter and other contacts you can find [here](https://thefuzzstone.github.io/).

---

I don't want to make this post too big, so if you have any additional questions feel free to ask them here.

**Comment by DiagonalArg** [comment]
@TheFuzzStone - checking the #feature-requests channel in keybasefriends, I see these people are admins:

adamjspooner
ayoubd
chris
cjb
mikem
misteele
patrick
strib

Why don't you try pining one of them directly?

**Comment by TheFuzzStone** [comment]
@DiagonalArg,

> Why don't you try pining one of them directly?

> mikem

I wrote to this admin (@mikem), he advised me to write @chris, but because I have reset my profile in 2017, for some reason I can not write him. 

Then I wrote to admin @mikem again, sent him a screenshot:

![paste](https://user-images.githubusercontent.com/16173361/80452225-4e591180-8915-11ea-9190-1ab4d8773296.png)


I asked him to contact me. @mikem did not answer me anymore.

I'll send a link to this issue to the #feature-requests chat room. 🤷🏻‍♂️
